api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/TrainableLayer.kt

```diff
@@ -0,0 +1,32 @@
+package org.jetbrains.kotlinx.dl.api.core.layer
+
+import org.tensorflow.op.core.Variable
+
+public interface TrainableLayer : ParametrizedLayer {
```

Comment:
added ParametrizedLayer/TrainableLayer interfaces, so only the layers that actually have parameters and/or can be trained will extend it